processor/index.d.ts

```diff
@@ -4,11 +4,9 @@ import { TranslationJSON, TranslationFunction } from '../create-i18n/index.js'
 
 export interface Processor<Key extends string = string> {
   from: ReadableAtom<Key>
-  (input: Record<Key, TranslationJSON>): TranslationFunction<
-    [],
-    // @ts-ignore
-    Input[keyof Input]
-  >
+  <Input extends Record<Key, TranslationJSON>>(
+    input: Input
+  ): TranslationFunction<[], Input[keyof Input]>
```

Comment:
Fixed this `// @ts-ignore`

Needs to be properly tested

Reply:
I have this comment because of TS error which affect `check-dts` but do not affect real TS typing.